sitemap/sitemap_test.go

```diff
@@ -95,7 +96,7 @@ func initializeRepo() (*SitemapRepository, error) {
 	utils.TruncateTables(testDB)
 	utils.PrepareTablesData(testDB)
 
-	careerRepo = jobs.New(testDB, templateFS)
+	careerRepo = jobs.New(testDB, templateFS, emailRepo)
```

Comment:
Here emailRepo can be nil, because no need of it in sitemap

Reply:
Okay change is done please review @cp-sumi-k